src/Microsoft.DotNet.Interactive/Parsing/DirectiveNode.cs

```diff
@@ -568,20 +568,33 @@ public async Task<IReadOnlyList<CompletionItem>> GetCompletionsAtPositionAsync(i
 
             case DirectiveParameterValueNode directiveParameterValueNode:
             {
-                if (directiveParameterValueNode.Parent is DirectiveParameterNode pn &&
-                    currentToken is not { Kind: TokenKind.Whitespace } &&
-                    pn.TryGetParameter(out var parameter))
+                if (directiveParameterValueNode.Parent is DirectiveParameterNode pn)
                 {
-                    var completions = await parameter.GetValueCompletionsAsync();
-                    return completions;
+                    if (currentToken is not { Kind: TokenKind.Whitespace })
+                    {
+                        if (pn.TryGetParameter(out var parameter))
+                        {
+                            var completions = await parameter.GetValueCompletionsAsync();
+                            return completions;
+                        }
+                    }
+                    else
+                    {
+                    }
```

Comment:
Looks like the `else` be removed. Also, if there is no `else` clause, looks like we can flip the nested `if` statement back to a single `if` statement with a compound condition check.